src/constants/extensions.ts

```diff
@@ -585,6 +585,7 @@ export const supportedExtensionsConfig: ExtensionConfig[] = [
       ColonyRole.Architecture,
       ColonyRole.Funding,
     ],
+    neededColonyVersion: 16,
```

Comment:
We are already maintaining this for all extensions and all colony versions: https://github.com/JoinColony/colonyJS/blob/main/packages/core/src/versions/helpers.ts#L47

I think we should have that in one single place as opposed to copied everywhere.
Let me know if I can help integrating this.

Reply:
Oh nice! Yeah that's much better. I see multisig isn't there though, what's the best way of going about this? If we added multisig there is it easy to get it into this PR, or is it worth creating a cleanup ticket to add that to colonyJS and update this part of the cdapp?

Reply:
Finally got it to work :) Thanks for all your help with it today!